UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -225,23 +236,43 @@ void notConnected() {
         }
     }
 
+    /*
+     * helper method to check if stop request evaluates as true.
+     * if stop request has is true then the stop status lights up and is made visible; otherwise
+     * its color is set to white to appear invisible.
+     */
+
+    public void checkStopRequest() {
+        sharedPreferences = getSharedPreferences(KEY_STOP, Context.MODE_PRIVATE);
+
+        if (sharedPreferences.getBoolean(KEY_STOP, false)) {
+            mStopRequest.setBackgroundColor(Color.parseColor("#B33701"));
+        } else {
+            mStopRequest.setBackgroundColor(Color.WHITE);
+        }
+    }
+
+    /*
+     * helper method to check if accessibility request evaluated as true.
+     * if accessibility request has is true then the stop status and accessibility symbols
+     * light up and is made visible; otherwise their colors are set to white to appear invisible.
+     */
+
+    public void checkAccessibilityRequest() {
+        sharedPreferences = getSharedPreferences(KEY_HANDICAP, Context.MODE_PRIVATE);
+
+        if (sharedPreferences.getBoolean(KEY_HANDICAP, false)) {
+            mAccessibilityRequest.setColorFilter(Color.parseColor("#008080"));
+        } else {
+            mAccessibilityRequest.setColorFilter(Color.WHITE);
+        }
+    }
+
     void drive(int throttleSpeed, int steeringAngle, String actionDescription) {
         notConnected();
         Log.i(TAG, actionDescription);
-        if(throttleSpeed > speed + 5 || throttleSpeed < speed - 5 || throttleSpeed == 0){
-            speed = throttleSpeed;
-            mMqttClient.publish(THROTTLE_CONTROL, Integer.toString(throttleSpeed), QOS, null);
-        }
-        if(steeringAngle > 10 && angle <= 0){
-            angle = 30;
-            mMqttClient.publish(STEERING_CONTROL, Integer.toString(angle), QOS, null);
-        }else if(steeringAngle < -10 && angle >= 0){
-            angle = -30;
-            mMqttClient.publish(STEERING_CONTROL, Integer.toString(angle), QOS, null);
-        }else if (steeringAngle <= 10 && steeringAngle >= -10 && angle != 0){
-            angle = 0;
-            mMqttClient.publish(STEERING_CONTROL, Integer.toString(angle), QOS, null);
-        }
```

Comment:
Why did you deleted this part from the code ? (Just wondering)

Reply:
I removed it upon request. @gusaxtcha and @iachella wanted to make their own PR to solve the latency issue, which is dependent on this code. 

Please see the conversation below :)
![C96CF98E-9CC7-4551-8B8D-E402A50B10D4](https://user-images.githubusercontent.com/38304351/119268226-5061d280-bbf2-11eb-96bc-2421cbd2ba87.jpeg)

Reply:
Ok good! Thanks for the clarification =)